frontend/src/components/bucket/BucketPermission.vue

```diff
@@ -30,17 +31,32 @@ const updateBucketPermission = (value: any, userId: string, permCode: string) =>
 const removeBucketUser = (userId: string) => {
   bucketStore.removeBucketUser(props.bucketId, userId);
 };
+
+const displayBucketPermissionUser : any = ref(false);
```

Comment:
Should this be of type boolean?

Reply:
This is the old code. Should I change it?